week_6/week_6_quiz.md

```diff
@@ -75,6 +75,18 @@ How does structured data annotation help the research process? (Select all that
 - [ ] Can give a better understanding about how data were acquired and processed
 - [ ] Can help generate research hypotheses
 
+<details>
+<summary>Click to see answer</summary>
+
+- [x] Can make filtering information within and between datasets easier
+- [x] Can reduce ambiguity about the measures recorded in a dataset
+- [x] Can give a better understanding about how data were acquired and processed
+- [x] Can help generate research hypotheses
```

Comment:
I noticed in #11 and #8 that other TAs were just pasting the answers that should be selected instead of reproducing the question with the appropriate answers checked off. I think your approach might be slightly more readable, but since the other PRs have already been merged it might be best to use their style for consistency.

Reply:
got it, switched to their style.